### DISCUSSION TITLE
Provide a streamlined system?

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [actions/virtual-environments#3591] [Installed software]
As my title says, I hope actions can provide some lightweight system environment.
A clean system allows developers to more easily configure an environment that suits their habits.
Or some special environment.
And I believe that few projects need to use so many different language environments at the same time.
Why not save precious space and provide it to users
(Although 14g has met the needs of most projects
But isn’t the more space the better? XD
The minimum storage space required for Ubuntu server is _2gb_
Imagine: a free compilation server! _30g+_ free space!

**Comment by Trolldemorted** [comment]
I have the problem that the installed packages interfere with my tests, because software that I am building is already preinstalled.

Is there any workaround that I can use to get a clean runner environment?
